### ISSUE TITLE
phrasal-verbs.pl

### ISSUE THREAD

**Issue by gargaamel** [gargaamel/polskie-ciasteczka-rodo-zgloszenia#113]
ciasteczka

![ss247](https://user-images.githubusercontent.com/44813765/49753936-1700ab00-fcb5-11e8-8916-c849c8e3e68d.png)

<details not-open>
<summary>Komunikat występuje nawet przy włączonych poniższych dodatkowych listach (i jak coś to nie zgłaszam komunikatów, które występują po ich wyłączeniu, bo mnie one nie interesują):</summary>

- dodatek I don't care about cookies
- lista Fanboy’s Cookie List

</details>

**Comment by gargaamel** [comment]
Błąd - brak przewijania, ale i tak to zgłoszenie nie kwalifikuje się, bo komunikat nie jest zamykany, więc zamykam, a filtry do usunięcia.

**Comment by krystian3w** [comment]
To może dodam w suplemencie, nie zauważyłem braku przewijania.

**Comment by krystian3w** [comment]
Na razie spróbuję odtworzyć błąd z brakiem przewijania i potem pomyślę.

**Comment by krystian3w** [comment]
Jaka rozdziałka monitora?

1366x768 z F11 i już nie ma przewijania bo strona krótka.

**Comment by krystian3w** [comment]
Kategorie się ładnie przewijają:

[http://phrasal-verbs.pl/category/g](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http://phrasal-verbs.pl/category/g)

**Comment by gargaamel** [comment]
Racja, brak przewijania nie był spowodowany błędem w filtrach, tylko po prostu ukryty komunikat `cookies `spowodował, że strona zrobiła się zbyt krótka w pionie i dlatego pasek się nie pokazuje.

**Comment by krystian3w** [comment]
To ja taki wytnę jak jest dobry uchwyt CSS, oszczędzam puste "P" i stopki bez zamykacza tekstu o cookies.

**Comment by krystian3w** [comment]
Nie widzę w tym czegoś bardzo złego.

**Comment by krystian3w** [comment]
W sumie dla mnie teraz było by trudne wywalić to kilka co też nie miało zamykacza a miało uchwyt lub bardzo się wyróżniało na tle swoich elementów...

Problem jaki widzę, to gdy nie sprawdzamy tekstu w elemencie czy ma `cookie` / `rodo`.

**Comment by gargaamel** [comment]
Czyli czasami takie bez zamykacza kwalifikują się, gdy są zbyt rzucające się w oczy, szkoda, że dopiero teraz o tym mnie poinformowano, bo tych, których nie zgłosiłem, to już nie zgłoszę, bo nie pamiętam które to były strony mogę co najwyżej od tej chwili takie zgłaszać...

**Comment by krystian3w** [comment]
OK, to by nie komplikować tylko rzucające się w oczy, reszta jak w AdGuardzie, do filtrów własnych jak często przeglądamy (nie całe 10 milionów musi mieć to wycięte jak nawet 1 raz nie wejdą).

:+1:
